src/calcium/qqbar.jl

```diff
@@ -767,7 +767,7 @@ Equivalently, $\operatorname{csgn}(x) = x / \sqrt{x^2}$ except that the value is
 at zero. The value is returned as a Julia integer.
 """
 function csgn(a::QQBarFieldElem)
-  return QQBarFieldElem(Int(ccall((:qqbar_csgn, libflint), Cint, (Ref{QQBarFieldElem}, ), a)))
+  return QQBarFieldElem(Int(@ccall libflint.qqbar_csgn(a::Ref{QQBarFieldElem})::Cint))
```

Comment:
Could be simplified now
```suggestion
  return QQBarFieldElem(@ccall libflint.qqbar_csgn(a::Ref{QQBarFieldElem})::Cint)
```